source/help/00README.md

```diff
@@ -167,11 +167,12 @@ sources:
     usage: append
 ```
 
-Note: as long as there is no other tag, the usage: `toplevel` is not necessary. Generated 00README files will contain it, but parsing allows for leaving out the toplevel specification.
+Note: as long as there is no other tag, the usage: `toplevel` is not necessary. Generated `00README` files will contain it, but parsing allows for leaving out the toplevel specification.
```

Comment:
Suggestion: Double colons in one sentence are a challenge. In this case, the second colon after "usage" is unnecessary and can be removed.